config/locales/en.yml

```diff
@@ -2112,6 +2112,7 @@ en:
     helper:
       time_future: "Ends in %{time}."
       until_login: "Active until the user logs in."
+      time_future_and_until_login: "Expires in %{time} and requires the user to login."
```

Comment:
The text is really not elegant, but the difference is that in the first case the block finishes when time has expired, in the later the block ends when the time has expired and the user has logged in. 

It is just that "Ends when the block has expired in %{time} and when the user has logged in." seems to be even more clumsy. Let me think a bit more about the wording.